### PR TITLE
[12.0] [MIG] project_task_material

### PR DESCRIPTION
**Pull request by anandkansagra**
Issue [Reference](https://github.com/OCA/project/issues/435)

### CODE REVIEW

File: project_task_material/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (alexey-pelykh) [inline project_task_material/__init__.py:3]: Please add license notice

File: project_task_material/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Copyright 2016 - Tecnativa - Vicent Cubells
+# Copyright 2017 - Tecnativa - David Vidal
+# Copyright 2018 - Brain-tec AG - Carlos Jesus Cebrian
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Project Task Material",
+    "summary": "Record products spent in a Task",
+    "version": "12.0.1.0.0",
+    "category": "Project Management",
+    "author": "Daniel Reis,"
```
Comment (alexey-pelykh) [inline project_task_material/__manifest__.py:12]: Please add `website` as per https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L2

File: project_task_material/README.rst [hunk 3]
```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (alexey-pelykh) [inline project_task_material/README.rst:1]: Please split into `readme` as per https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

File: project_task_material/views/project_view.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
```
Comment (alexey-pelykh) [inline project_task_material/views/project_view.xml]: License and copyright notice would be great, yet not required

File: project_task_material/tests/common.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+from odoo.tests.common import TransactionCase
+
+
+class TestProjectCases(TransactionCase):
+    """Prepare data to test the module."""
+
+    def setUp(self):
+        """Create user, task, project as well as refre action of the user."""
+        super(TestProjectCases, self).setUp()
```
Comment (alexey-pelykh) [inline project_task_material/tests/common.py:12]: Not required, yet just `super()` is ok

File: project_task_material/tests/__init__.py [hunk 6]
```diff
@@ -0,0 +1 @@
+from . import test_create_material_lines
```
Comment (alexey-pelykh) [inline project_task_material/tests/__init__.py:3]: License notice needed

File: project_task_material/models/__init__.py [hunk 7]
```diff
@@ -0,0 +1 @@
+from . import project
```
Comment (alexey-pelykh) [inline project_task_material/models/__init__.py:3]: License notice needed

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

File: project_task_material/tests/common.py (hunk 5 above)
Comment (nikul-serpentcs) [reply project_task_material/tests/common.py:12]: > Not required, yet just `super()` is ok

@alexey-pelykh It's not blocking.

Comment (alexey-pelykh) [reply project_task_material/tests/common.py:12]: Totally agree, thus not required

**Comment by pedrobaeza** [comment]
Please add #456 in this PR, and rewrite your commit message to be `[MIG] project_task_material_stock: Migration to 12.0` or similar, but not to be `[ADD] ...`, as this is not an addition.

**Comment by pedrobaeza** [comment]
Sorry, #456 is for `project_task_material_stock`, so it doesn't apply here.

**Review by alexey-pelykh — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**